torch/_inductor/codegen/triton.py

```diff
@@ -2099,6 +2099,10 @@ def codegen_kernel(self, name=None):
                     from torch._inductor.ir import TileHint
                     from torch._inductor.triton_heuristics import AutotuneHint, {heuristics}
                     from torch._inductor.utils import instance_descriptor
+                    try:
+                        from triton.compiler.compiler import AttrsDescriptor
+                    except ImportError:
+                        pass  # ignore
```

Comment:
Let's check the Triton version either include or exclude this to keep code cleaner.